src/java/com/sarality/app/keyboard/KeyboardActionListener.java

```diff
@@ -0,0 +1,50 @@
+package com.sarality.app.keyboard;
+
+import android.inputmethodservice.KeyboardView;
+
+/**
+ * Listens to keyboard actions and updates the editable text as well as the keyboard that is visible.
+ *
+ * @author abhideep@ (Abhideep Singh)
+ */
+public class KeyboardActionListener implements KeyboardView.OnKeyboardActionListener {
```

Comment:
For now it does nothing but delegates to the keyboard changer. 

I am sending you a few CLs in flights that we call this to switch between flights and normal keyboard. We'll add to this to support number and symbols keyboard.

We'll also add a KeyboardOutputComposer that will allow us to compose the text as output by the various keyboards. The Listeners delegates to the composer as well.